.gitlab-ci.yml

```diff
@@ -136,19 +136,28 @@ test:build-output:
 deploy:sls:branch:
   <<: *tmpl_branch
   <<: *tmpl_deploy
```

Comment:
do we need tmp_deploy if we are overwriting with `script`? i guess its for the `dependencies` but maybe this should be copied across, otherwise it kinda implies the reuse of more functionality than it really does

Reply:
The usage of these templates is quite questionable but this deploy one is the worst. No job uses it fully, either replacing the script or dependencies. It probably makes sense to just remove it. 🤷🏻‍♂️